src/framework/mpe/mpetypes.h

```diff
@@ -294,17 +294,17 @@ constexpr dynamic_level_t DYNAMIC_LEVEL_STEP = 5 * ONE_PERCENT;
 
 enum class DynamicType {
     Undefined = -1,
-    ppppppppp = MIN_DYNAMIC_LEVEL,
-    pppppppp = 5 * ONE_PERCENT,
-    ppppppp = 10 * ONE_PERCENT,
-    pppppp = 15 * ONE_PERCENT,
-    ppppp = 20 * ONE_PERCENT,
-    pppp = 25 * ONE_PERCENT,
-    ppp = 30 * ONE_PERCENT,
-    pp = 35 * ONE_PERCENT,
-    p = 40 * ONE_PERCENT,
-    mp = 45 * ONE_PERCENT,
-    Natural = 50 * ONE_PERCENT,
+    pppppppppp = MIN_DYNAMIC_LEVEL,
```

Comment:
I don't love this change, but I wasn't just how to do this better without making the surface of the change larger.

Because you still want each dynamic level increase to be the same, you either need to make the step larger (100 / 19 in this case), or add another "fake" dynamic level.  Since the dynamics seemed to already cover unrealistic markings, I added a new dynamic below "ppppppppp", to keep the step interval to a cleanly divisible 100/20 =  5.